db/schema.rb

```diff
@@ -244,7 +244,7 @@
     t.string "email", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.boolean "operator", default: false
+    t.boolean "operator", default: false, null: false
```

Comment:
Similarly curious why this changed?

Reply:
My best guess is that the default behavior for boolean columns changed between Rails 6 and 7; but I don't think it's significant enough to worry about.